### DISCUSSION TITLE
The dragon can't go to the portal

### DISCUSSION THREAD

**Discussion by fredypawww** [PurpurMC/Purpur#1330] [__Dismissed__] [answered]
Hello. I wanted to send a dragon into the world (as shown in the video https://www.youtube.com/watch?v=g_-_oSr4l_w ), but the dragon cannot enter the portal/gateway, even when entities-can-use-portals is enabled. It works correctly in the vanilla version. Could you fix this mechanic?

**Comment by encode42** [comment]
Does it work on Paper?

**Reply by fredypawww** [reply]
no

**Reply by katekerllenevich** [reply, marked as answer]
Then this is an upstream issue, and should be opened with Paper
